docs/lib/datastore/fragments/flutter/data-access/delete-snippet.md

```diff
@@ -1,5 +1,13 @@
-To delete an item, simply pass in an instance to `Amplify.DataStore.delete()`.  Below, we query for an instance with an `id` of `"123"`, and then delete it, if found:
```

Comment:
Why delete this?  It is not STRICTLY necessary to query for an instance first, so this first sentence adds a bit of clarity.

Reply:
This first sentence was repeated twice, at least when running in my local instance that's why I removed it, totally agree that it's important.